### ISSUE TITLE
No response

### ISSUE THREAD

**Issue by bit4lifeonline** [thewhiteh4t/seeker#21]
It works in Termux when i upgrade the system!but when it run and clicked the URL no response....
[!] Keyboard Interrupt.
AH00558: httpd: Could not reliably determine the server's fully qualified domain name, using 127.0.0.1. Set the 'ServerName' directive globally to suppress this message

**Comment by thewhiteh4t** [comment]
this message is fine its not an error, can you share install.log present in seeker directory

**Comment by bit4lifeonline** [comment]
[install.log](https://github.com/thewhiteh4t/seeker/files/2397521/install.log)
here it is

**Comment by bit4lifeonline** [comment]
yes,already click the link and click "continue" but no any response in Termux.

**Comment by thewhiteh4t** [comment]
Open the link in firefox and press F12 to open console, see if you find any errors

**Comment by thewhiteh4t** [comment]
Tested with fresh termux, its working for me

**Comment by bit4lifeonline** [comment]
I try it later,thx for all.

**Comment by bit4lifeonline** [comment]
Connecting with address: 18.216.53.253

Disconnect with reason: Failed to establish connection (Attempt to connect to host timed out)

**Comment by thewhiteh4t** [comment]
What is this?

**Comment by bit4lifeonline** [comment]
is that my android phone network connection problem?

**Comment by thewhiteh4t** [comment]
Where are you getting this error?

**Comment by bit4lifeonline** [comment]
vctim terminal

**Comment by thewhiteh4t** [comment]
Victim terminal?
I think you should see my youtube demo in readme

**Comment by bit4lifeonline** [comment]
I already seen that,it work in kali very well.

**Comment by thewhiteh4t** [comment]
I have two devices and i reinstalled termux on both, cloned seeker, installed it, and its working in both

**Comment by bit4lifeonline** [comment]
but Termux in Android can't receive the response.

**Comment by thewhiteh4t** [comment]
Is the page completely loaded?

**Comment by bit4lifeonline** [comment]
yes, I try to change the phone and try again
